Transcelerator/TxlQuestions.xml

```diff
@@ -33804,6 +33932,15 @@
         <Questions scrref="JHN 19.24" startref="43019024" endref="43019024">
           <Q>What is the reason that the soldiers divided Jesus's clothing?</Q>
           <A>They divided Jesus's clothing so that the [Old Testament] scripture was fulfilled. (24)</A>
+          <Alternative source="TomBogle">What is the reason that the soldiers cast lots among themselves for Jesus' clothing?</Alternative>
+          <Alternative source="TomBogle">What is the reason that the soldiers cast lots to divide Jesus' clothing among themselves?</Alternative>
+          <Alternative source="TomBogle">What is the reason that the soldiers cast lots to decide which of them would get to take parts of Jesus' clothing?</Alternative>
+          <Alternative source="TomBogle">What is the reason that the soldiers divided Jesus's clothing among themseleves?</Alternative>

```

Comment:
typo (themseleves) x2

Reply:
Fixed. You inspired me to write a spell-checker. Caught several other errors!